foundry.toml

```diff
@@ -99,14 +99,17 @@
   wrap_comments = true
 
 [rpc_endpoints]
-  arbitrum = "https://arbitrum-mainnet.infura.io/v3/${API_KEY_INFURA}"
-  avalanche = "https://avalanche-mainnet.infura.io/v3/${API_KEY_INFURA}"
+  arbitrum = "${ARBITRUM_RPC_URL}"
+  arbitrum_sepolia = "https://arbitrum-sepolia.blockpi.network/v1/rpc/public"
+  avalanche = "${AVALANCHE_RPC_URL}"
   base = "https://mainnet.base.org"
+  base_sepolia = "https://sepolia.base.org"
   bnb = "https://bsc-dataseed.binance.org"
   gnosis = "https://rpc.gnosischain.com"
   localhost = "http://localhost:8545"
-  ethereum = "${RPC_URL_MAINNET}"
-  mainnet = "${RPC_URL_MAINNET}"
-  optimism = "https://optimism-mainnet.infura.io/v3/${API_KEY_INFURA}"
-  polygon = "https://polygon-mainnet.infura.io/v3/${API_KEY_INFURA}"
-  sepolia = "https://sepolia.infura.io/v3/${API_KEY_INFURA}"
+  mainnet = "${MAINNET_RPC_URL}"
```

Comment:
I still think it is confusing to be subjective. For example, we are using an URL directly here for `base` and `scroll` while for others, we have RPC URL loading from `.env.deployment`. How about the following:

- Keep all variables in `.env.deployment.example`
- Keep only what is required for development in `.env.example`
- Have the same env variables names
- We can even set values to public URL directly in the environment example files.

wdyt?

Reply:
> we are using an URL directly here for base and scroll while for others

because the RPC urls used directly here are public by the chain org, [base](https://docs.base.org/docs/network-information/), [bnb](https://www.binance.com/en/square/post/3613468649713), [scroll](https://docs.scroll.io/en/developers/developer-quickstart/).
in case of mainnet (and the other) it requires to have a third party providere (e.g. infura) to use a rpc url

> Keep all variables in .env.deployment.example

we are doing this for the RPCs that are not used in devolpment

> Keep only what is required for development in .env.example

we are doing this, we keep only `MAINNET_RPC_URL` in `.env.example` (which is used for fork tests)

> Have the same env variables names

we are doing this, what var is not having the same name?

> We can even set values to public URL directly in the environment example files 

i don't think this is a good idea, the .env.example is ignored by the system, you would still need to create an .env file to place them

Reply:
Agree to all your points. 

> we are doing this, what var is not having the same name?

It was different, but you have made them all same in this PR which is what I was suggesting.

I feel I may have not been able to explain it properly. I created a commit to show what I have in my mind: https://github.com/sablier-labs/v2-core/commit/8a02d6bdb12d9afc4e24197792ec0ba7a62e0035

Let me know your thoughts.

Reply:
> We can even set values to public URL directly in the environment example files

one more point against this is that you need to declare the RPC URL explicitly in `.env.deployment` file:

```
# foundry.toml file
base = "${BASE_RPC_URL}"
# .env file
BASE_RPC_URL="https://mainnet.base.org"
```

vs

```
# foundry.toml file
base = "https://mainnet.base.org"
```

Reply:
Yes thats what I was suggesting but since its a public endpoint, we can even include it in `.env.deployment.example`

Reply:
is there any other reason to place them in `.env` file other than consistency? being consistent here means to repeat ourselves, see [comment](https://github.com/sablier-labs/v2-core/pull/973#discussion_r1681021072).

imo, storing actual public env vars in the `.env.example` loses the purpose of a `.env` file:

storing development-only, _non-sensitive_ versions of your service's environment variables

Reply:
As resolved over the slack, lets keep it you way.